cmd/root.go

```diff
@@ -73,7 +74,16 @@ func initConfig() {
 
 	viper.AutomaticEnv()
 
-	if err := viper.ReadInConfig(); err == nil {
-		fmt.Fprintln(os.Stderr, "Using config file:", viper.ConfigFileUsed())
+	err := viper.ReadInConfig()
+
+	var notFound *viper.ConfigFileNotFoundError
+	switch {
+	case err != nil && !errors.As(err, notFound):
+		cobra.CheckErr(err)
+	case err != nil && errors.As(err, notFound):
+		// The config file is optional, we shouldn't exit when the config is not found
+		_, _ = fmt.Fprintln(os.Stderr, "Config file not found")
```

Comment:
Nowhere else in the code base do we use the form `_, _ = fmt.*`, so for consistency go ahead and just call the `fmt.Fprintln()` function.